Gemfile

```diff
@@ -17,3 +17,4 @@ gem 'json', '~> 1.8.2'
 gem 'git', '~> 1.2.6'
 gem 'trollop', '~> 2.0'
 gem 'coderay', '~> 1.1.0'
+gem 'rb-inotify', '~> 0.9'
```

Comment:
No, it failed on my Fedora F22. But could you please do a test for me? Can you build this on your Docker with `gem 'rb-inotify', '~> 0.9'`? If so, I would prefer to leave it (since it would make both of us happy)